### ISSUE TITLE
[ZOOM]: Foresee survey modal can't be easily collapsed at 400% zoom

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5628]
## Description

The Foresee modal window that appears on new user screens cannot be easily closed at 400% zoom when browsers are set at 1280px. This might be a more global issue, and I'm happy to reassign as `sitewide` if necessary. Screenshots attached below.

## Type of Issue
Template

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Foresee modal can be interacted with or dismissed at 400% zoom

## Environment

* Chrome
* MacOS Mojave
* Server destinations:
  * https://www.va.gov/pittsburgh-health-care/locations/fayette-county-va-clinic/
  * Any other Pittsburgh medical center pages

## Steps to Recreate

1. Clear your cache of all cookies, cached files, images, everything, for all time, to ensure the Foresee survey launches
1. Set browser width to exactly 1280px
1. Press `Ctrl + Plus` to zoom in to 400%
1. Enter a server destination from above
1. Verify the Foresee modal close button cannot be seen on screen

## Possible Fixes (optional)
Coordinate with Foresee to find a way to update CSS, or make local modifications if we can do so.

## WCAG or Vendor Guidance (optional)

* [Understanding Success Criterion 1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-06 at 5.01.56 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/fbf39031-6afc-40b2-9685-0218eb19a053)

**Comment by davidconlon** [comment]
@1Copenut I'm having problems duplicating.  Can you walk us through this one?  I'm also thinking this may be a site-wide issue.
Following on that, do we have standard zoom levels that we test to documented somewhere? (apologies if I didn't see this in the onboarding docs)

**Comment by 1Copenut** [comment]
@davidconlon Hi Dave. I wondered if this one might be tricky to reproduce. Happy to run through it with you next week and see if we can spot the issue again. Here's a deep link to my accessibility best practices document. I'm making a few edits, but the section on zooming is solid: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#zoom-to-400

**Comment by mmiddaugh** [comment]
If this is occurring on Pittsburgh, I'm wondering if it is also an issue on Facility Locator, @1Copenut

**Comment by davidconlon** [comment]
@1Copenut is this unique to FL/VAMC or is this something that is site-wide that uses Forsee?  Can you test?

**Comment by 1Copenut** [comment]
@davidconlon Yes, I'll retest this as soon as I've got a free moment. If it's a wider issue, I'll reassign this to the Tools team and label accordingly.

**Comment by 1Copenut** [comment]
@davidconlon I just revisited this and determined it was a sitewide issue with the overlay. I'll reassign it to the right team.

**Comment by 1Copenut** [comment]
@brandonrapp && @nedierecel && @joanneesteban I originally sent this to the Facilities team because I spotted the overlay in a content audit. It turns out this was a global issue with the overlay, so reassigning to you all. Please let me know if I should send it elsewhere. Thank you.

**Comment by joanneesteban** [comment]
Thanks for sending over, @1Copenut . Unassigning @nedierecel since this is a ForeSee issue. 

At the outset, this is probably a global ForeSee product issue, but I'll take a closer look and let you know if I need Front-End support @brandonrapp

**Comment by joanneesteban** [comment]
Sent a support ticket to our ForeSee contact & will await response.

**Comment by joanneesteban** [comment]
@1Copenut Hi! I contacted ForeSee and they appear to have fixed the issue on their end. I received the following message:

> When zoomed in 400 percent I am still able to scroll to the top right and click the 'x' to close the survey. 

I was able to do the same on my end. Can you confirm on your end? I'll close the issue with ForeSee when I get your confirmation. Thanks!

**Comment by 1Copenut** [comment]
@joanneesteban I tried the Foresee modal out with Firefox, since Chrome's my daily driver. I was able to recreate the issue when I had my browser zoomed it to 300% at 1280px width, so I'm not sure if the issue is closed or not. I was looking on Staging rather than prod. Can you send me a screen shot of your test? That should give us a definitive answer.

Thank you!

---
![Screen Shot 2020-05-06 at 12 25 48 PM](https://user-images.githubusercontent.com/934879/81209231-dedac600-8f95-11ea-8f1f-200864f27d17.png)

**Comment by joanneesteban** [comment]
Hey @1Copenut , you're right. I tried recreating again in Chrome and prod and I'm still getting this issue. I'll follow up with ForeSee.

![image](https://user-images.githubusercontent.com/48964695/81223445-fcb72380-8fb3-11ea-8989-cebd164b6cc8.png)

**Comment by 1Copenut** [comment]
@joanneesteban Thank you for confirming. LMK if I can help retest later.

**Comment by joanneesteban** [comment]
@1Copenut 

ForeSee sent [this video (attaching from Slack)](https://dsva.slack.com/archives/CKCRMLGAC/p1588875422332400). Let me know if this works as the desired behavior and if you'd like for them to test further in Staging as well, and I can give them the go-ahead to push this update live.

**Comment by 1Copenut** [comment]
@joanneesteban This looks good to me!

**Comment by joanneesteban** [comment]
Hey @1Copenut ! I received this message from ForeSee support:

> All requested changes as discussed previously have been reviewed and pushed out to the staging container (sitekey: va-gov) for you to test. For reference, the change currently in your staging container is:
> - Update invitation to scroll when the browser is fully zoomed and the full invite cannot be displayed on the screen
> 
> Although we test for expected behavior in a local environment, we'd like you to have the opportunity to ensure that our product interacts as desired with your website. Our documentation on testing can be found here: https://developer.foresee.com/docs/testing-web-sdk.
> 
> Once you're able to verify that the code properly runs within your testing environment, please reply to this email to let us know that you are ready to move the contents of your staging container to your production container. Without your approval, these changes will not be pushed live. If you have any concerns moving forward, please reach out to support@foresee.com and we'll be happy to address them.

Is this test something you would be able to help me out with, or should I tag someone else for support?

**Comment by 1Copenut** [comment]
@joanneesteban I retested this with Firefox at 300%, and it was behaving correctly. If you're getting good results with Chrome too, let's move this forward.

**Comment by joanneesteban** [comment]
@1Copenut same here. Will let ForeSee know!

**Comment by joanneesteban** [comment]
ForeSee pushed to prod last month. Closing this issue!
